yarn.lock

```diff
@@ -179,6 +196,29 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/core@npm:^7.22.0, @babel/core@npm:^7.22.9":
```

Comment:
There seems to be some duplicate deps (e.g. `@babel/core` here). I'll try to run `yarn dedupe` to see if it helps.